backend/slepr-master.xml

```diff
@@ -79,8 +79,8 @@
 			</sld:ElectronicAddress>
 		</sld:Provider>
 		<sld:ServiceListOffering regulatorListFlag="false">
-		<sld:ServiceListName>DVB-I Reference Client service list</sld:ServiceListName>
-		<sld:ServiceListName xml:lang="zh">DVB-I 参考客户端服务列表</sld:ServiceListName>
+		<sld:ServiceListName>DVB-I Reference Client default service list</sld:ServiceListName>
```

Comment:
What about adding "New users should choose this list" or something like that? "default service list" sort-of implies that but people for who English isn't their first or second language may not pick up on that.